src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -52,22 +53,13 @@ def command_create_profile(
     _, storage_entry_point = get_entry_point_from_class(storage_cls.__module__, storage_cls.__name__)
     assert storage_entry_point is not None
 
-    if kwargs.pop('use_rabbitmq'):
-        broker_backend = 'core.rabbitmq'
-        broker_config = {
-            key: kwargs.get(key)
-            for key in (
-                'broker_protocol',
-                'broker_username',
-                'broker_password',
-                'broker_host',
-                'broker_port',
-                'broker_virtual_host',
-            )
-        }
+    broker_config = detect_rabbitmq_config()
+    broker_backend = 'core.rabbitmq' if kwargs.get('use_rabbitmq', False) else None
+
+    if broker_config is None:
+        echo.echo_warning('RabbitMQ server not found: configuring the profile without a broker.')
```

Comment:
Maybe here, we can inform the user that one can configure it manually with `verdi profile configure-rabbitmq`? Maybe in actually also in the case when it is successful. Also it seems to me that this information should be in the docs here https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/installation.html#rabbitmq-configuration

Reply:
Great points @agoscinski . We are still finalizing the exact behavior we want for creating profiles and configuring RMQ. I was planning on first finalizing all of that and then as the last step update the docs PR to reflect the final state.